Content.Server/StationEvents/Components/StationEventComponent.cs

```diff
@@ -42,6 +42,19 @@ public sealed partial class StationEventComponent : Component
     [DataField]
     public int ReoccurrenceDelay = 30;
 
+
+    /// <summary>
+    /// Alert level to set the station to when the event starts.
+    /// </summary>
+    [DataField]
+    public string? AlertLevel;
```

Comment:
use protoIds to validate these

Reply:
alert levels aren't prototypes, they're stringes

THe alertlevelprototypes actually contain the alert levels, its why they were strings in the original AlertLevelInterceptionRule